code/_core/obj/item/fluff/bikehorn.dm

```diff
@@ -18,7 +18,9 @@ obj/item/bikehorn/click_self(var/mob/caller)
 	INTERACT_DELAY(20)
 
 	var/turf/T = get_turf(src)
+  
 	play_sound('sound/items/bikehorn.ogg',T)
+
```

Comment:
Try to avoid empty commits to files in the future.

Reply:
was a merge conlfict, but ill try